mlir/test/Dialect/MIOpen/lowering_affix_params.mlir

```diff
@@ -2,4 +2,4 @@
 // RUN: mlir-miopen-driver -p --operation=conv2d_dummy -miopen-lowering -miopen-affine-transform -miopen-affix-params | FileCheck %s
 
 // CHECK: module {{.*}}
-// CHECK-NEXT: func @{{.*}}(%{{.*}}: memref<{{.*}}>, %{{.*}}: memref<{{.*}}>, %arg2: memref<{{.*}}>) attributes {block_size = {{.*}} : i32, grid_size = {{.*}} : i32, kernel}
+// CHECK-NEXT: func @{{.*}}(%{{.*}}: memref<{{.*}}>, %{{.*}}: memref<{{.*}}>, %arg2: memref<{{.*}}>) attributes {block_size = {{.*}} : i32, grid_size = {{.*}} : i32, kernel = 0 : i32}
```

Comment:
please supply unit test where `kernel_id != 0` and we see `miopen.conv2d_dummy` gets populated. probably it needs to be a separate test file.